src/OpenIDConnectClient.php

```diff
@@ -1131,35 +1110,16 @@ private function verifyRSAJWTsignature($hashtype, $key, $payload, $signature, $s
             '  <Modulus>' . b64url2b64($key->n) . "</Modulus>\r\n" .
             '  <Exponent>' . b64url2b64($key->e) . "</Exponent>\r\n" .
             '</RSAKeyValue>';
-        if (class_exists('\phpseclib3\Crypt\RSA', false)) {
-            $key = \phpseclib3\Crypt\PublicKeyLoader::load($public_key_xml)
-                ->withHash($hashtype);
-            if ($signatureType === 'PSS') {
-                $key = $key->withMGFHash($hashtype)
-                    ->withPadding(\phpseclib3\Crypt\RSA::SIGNATURE_PSS);
-            } else {
-                $key = $key->withPadding(\phpseclib3\Crypt\RSA::SIGNATURE_PKCS1);
-            }
-            return $key->verify($payload, $signature);
-        } elseif (class_exists('Crypt_RSA', false)) {
-            $rsa = new Crypt_RSA();
-            $rsa->setHash($hashtype);
-            if ($signatureType === 'PSS') {
-                $rsa->setMGFHash($hashtype);
-            }
-            $rsa->loadKey($public_key_xml, Crypt_RSA::PUBLIC_FORMAT_XML);
-            $rsa->setSignatureMode($signatureType === 'PSS' ? Crypt_RSA::SIGNATURE_PSS : Crypt_RSA::SIGNATURE_PKCS1);
-            return $rsa->verify($payload, $signature);
+
+        $key = PublicKeyLoader::load($public_key_xml)
```

Comment:
Suggestion of https://github.com/jumbojett/OpenID-Connect-PHP/issues/321

This can be removed:
https://github.com/jumbojett/OpenID-Connect-PHP/blob/6aae75b1c7b58d8d64a6d14877f77b8a13ab90f6/src/OpenIDConnectClient.php#L1143-L1146

And direct load the key
```suggestion
        $key = PublicKeyLoader::load($key)
```